app/web/src/components/FuncEditor/ActionDetails.vue

```diff
@@ -1,15 +1,35 @@
 <template>
   <div class="p-3 flex flex-col gap-2">
+    <span v-if="props.validationMessage" class="text-red-400">{{
+      props.validationMessage
+    }}</span>
     <h2 class="pt-4 text-neutral-700 type-bold-sm dark:text-neutral-50">
-      Kind of Action:
+      <SiCheckBox
```

Comment:
why not just use a radio or a dropdown? doesnt make sense IMO to have checkboxes if only one is selectable